src/3p-frame.js

```diff
@@ -102,16 +102,19 @@ export function getIframe(parentWindow, element, opt_type) {
 }
 
 /**
- * Allows listening for message from the iframe.
+ * Allows listening for message from the iframe. Returns an unlisten
+ * function to remove the listener.
+ *
  * @param {!Element} iframe
  * @param {string} typeOfMessage
  * @param {function(!Object)} callback Called when a message of this type
  *     arrives for this iframe.
+ * @return {function()}
  */
 export function listen(iframe, typeOfMessage, callback) {
```

Comment:
Since we are here, we may as well create `listenOnce` similar to the ones in `event-helper.js`?